packages/transformers/webextension/src/schema.js

```diff
@@ -438,6 +438,14 @@ export const MV3Schema = ({
       additionalProperties: false,
     },
     host_permissions: arrStr,
+    oauth2: {
```

Comment:
This [was supported in Manifest V2](https://developer.chrome.com/docs/extensions/mv2/tut_oauth/) and should therefore be moved to `commonProps` rather than `MV3Schema`. Also, make sure to use trailing commas to pass the linter.

Reply:
👍 thx
- move to `commonProps`
- add miss commas